src/platforms/hosted/cmsis_dap.c

```diff
@@ -262,15 +262,29 @@ static bool dap_init_bulk(void)
 }
 
 /* LPC845 Breakout Board Rev. 0 reports an invalid response with > 65 bytes */
-bool dap_init(void)
+bool dap_init(bool allow_fallback)
 {
 	/* Initialise the adaptor via a suitable protocol */
-	if (bmda_probe_info.in_ep && bmda_probe_info.out_ep) {
+	if (bmda_probe_info.in_ep && bmda_probe_info.out_ep)
 		type = CMSIS_TYPE_BULK;
-		if (!dap_init_bulk())
-			return false;
-	} else {
+	else
 		type = CMSIS_TYPE_HID;
+
+	/* Windows hosts may not have the winusb driver associated with v2, handle that by degrading to v1 */
+	if (type == CMSIS_TYPE_BULK) {
+		if (!dap_init_bulk()) {
```

Comment:
Because of how this is constructed, this can be combined into `if (type == CMSIS_TYPE_BULK && !dap_init_bulk())` just fine, allowing the whole lot to be de-indented by one.